From 16674ac75127b0e83691c6b1c9ce745e67ab58b6 Mon Sep 17 00:00:00 2001
From: Philip-Larsen-Donnelly
 <35666657+Philip-Larsen-Donnelly@users.noreply.github.com>
Date: Mon, 20 Sep 2021 21:55:47 +0200
Subject: [PATCH] build: correct patch PRs (#8771)

* build: correct patch PRs for CVE-2021-39179

* Update advisories.json
---
 .patches/security/advisories.json | 14 +++++---------
 1 file changed, 5 insertions(+), 9 deletions(-)

diff --git a/.patches/security/advisories.json b/.patches/security/advisories.json
index b64b260dd15..b682c02a12c 100644
--- a/.patches/security/advisories.json
+++ b/.patches/security/advisories.json
@@ -6,11 +6,7 @@
         "status": "FIXED",
         "pullRequests": [
           {
-            "prNumber": 11,
-            "target": "patch/2.34.7"
-          },
-          {
-            "prNumber": 4,
+            "prNumber": 14,
             "target": "patch/2.35.7"
           },
           {
@@ -18,7 +14,7 @@
             "target": "patch/2.35.8"
           },
           {
-            "prNumber": 3,
+            "prNumber": 16,
             "target": "patch/2.36.4"
           }
         ]
@@ -29,15 +25,15 @@
         "status": "FIXED",
         "pullRequests": [
           {
-            "prNumber": 6,
+            "prNumber": 7,
             "target": "patch/2.34.7"
           },
           {
-            "prNumber": 3,
+            "prNumber": 8,
             "target": "patch/2.35.8"
           },
           {
-            "prNumber": 2,
+            "prNumber": 9,
             "target": "patch/2.36.4"
           }
         ]
